src/Components/Job_info.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { FaAngleDoubleRight } from "react-icons/fa";
+
+type PropType = {
+  data: any[];
+  index: number;
+};
+
+const JobInfo: React.FunctionComponent<PropType> = ({ data, index }) => {
```

Comment:
lets think of it as if we want to re use to for other projects/feature and the data keys provided by backend might be different
so to handle that we can have specific props for it like title,company, dates, duties